provider/cmd/pulumi-resource-gcp-native/metadata.json

```diff
@@ -30666,76 +30075,6 @@
             ],
             "noDelete": true
         },
-        "gcp-native:pubsub/v1:Subscription": {
```

Comment:
Bummer to loose some of the pubsub resources. Curious if you see anything in the spec which might allow them to be modeled?
The rest seem like a good tradeoff though!

Reply:
FWIW, they didn't work. It's tracked in https://github.com/pulumi/pulumi-gcp-native/issues/38